CHANGES

```diff
@@ -34,6 +34,7 @@ Incompatible changes
 * #1857: latex: :confval:`latex_show_pagerefs` does not add pagerefs for
   citations
 * #4648: latex: Now "rubric" elements are rendered as unnumbered section title
+* #4983: html: The URL for the productionlist has been changed
```

Comment:
Maybe `anchors for productionlist tokens` rather than `URL for productionlist`?

Reply:
Good point